presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -62,6 +62,10 @@ statement
         ADD constraintSpecification                                    #addConstraint
     | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
         DROP CONSTRAINT (IF EXISTS)? name=identifier                   #dropConstraint
+    | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
+        ALTER COLUMN (IF EXISTS)? column=identifier SET NOT NULL       #alterColumnSetNotNull
+    | ALTER TABLE (IF EXISTS)? tableName=qualifiedName
+        ALTER COLUMN (IF EXISTS)? column=identifier DROP NOT NULL      #alterColumnDropNotNull
```

Comment:
pedantic, but according to the SQL 2016 spec, `COLUMN` is optional: https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#_11_12_alter_column_definition

Also, it seems `IF EXISTS` is not in the spec for `ALTER TABLE` or `ALTER COLUMN`.. It's probably fine to leave it though.

Reply:
Thanks for catching this. Fixed